frontend/src/views/LoginView.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import { StateContext } from '../State';
+
+const LoginView = () => {
+    const context = React.useContext(StateContext);
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+
+        let res = await fetch("http://localhost:8080/auth", {
+            method: "POST",
+            headers: {
+                "Accept": "application/json",
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify({
+                email: event.target.email.value,
+                password: event.target.password.value
+            })
+        })
+        
+        if (res.status === 200) {
+            let userID = await res.json()
+
+            res = await fetch(`http://localhost:8080/tickets/details?userID=${userID.id}`, {
+                method: "GET",
```

Comment:
The fetch API will always default to a GET method, so it isn't explicitly necessary to declare this as a GET.